src/main/java/net/education/kyivstar/config/SchemaAndTableCreator.java

```diff
@@ -0,0 +1,67 @@
+package net.education.kyivstar.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.DATABASE_PATH;
+import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.TABLES_PATH;
+import static net.education.kyivstar.services.util.Utils.readSqlScriptFromFileStream;
+
+public class SchemaAndTableCreator {
+    private static final Logger logger = LoggerFactory.getLogger(SchemaAndTableCreator.class);
+    private final EmbeddedMariaDbLifeCycle embeddedMariaDbLifeCycle;
+
+    public SchemaAndTableCreator(EmbeddedMariaDbLifeCycle embeddedMariaDbLifeCycle) {
+        this.embeddedMariaDbLifeCycle = embeddedMariaDbLifeCycle;
+    }
+
+    public void createDataBase() {
```

Comment:
The createDataBase() and createTables() methods have very similar logic. To avoid code duplication, you could refactor the common logic into a private method. This would make the code easier to maintain and reduce the chance of introducing bugs if the logic needs to be updated in the future.

Reply:
+